### PR TITLE
[twitter] Valid post body can be empty

### LINKED ISSUE #11458: Twitter: Image-only tweets don't show up in the Network page
I'm using the Twitter addon and it's been working as intended for the most part, except for image-only posts. When there's absolutely no text other than an image, the post never shows up in my timeline.

It even lead to a weird situation where I didn't receive an image-only tweet, but I received the reply from the same account giving credit for the picture.

Friendica conversation:
friendica-twitter-chaotic-nightclub

Original Twitter conversation: https://twitter.com/ClubPhotos_/status/1515782319776411661

### PR DESCRIPTION
**Pull request by MrPetovan**
Fixes https://github.com/friendica/friendica/issues/11458

### CODE REVIEW

File: twitter/twitter.php [hunk 1]
```diff
@@ -1194,6 +1197,13 @@ function twitter_do_mirrorpost(App $a, int $uid, $post)
 	return $datarray;
 }
 
+/**
+ * Fetches the Twitter user's own posts
+ *
+ * @param App $a
+ * @param int $uid
```
Comment (Quix0r) [inline twitter/twitter.php:1204]: I would not mind having an `@return void` being added here, later on I will add `: void` for methods returning nothing to be ready for newer PHP versions.

File: twitter/twitter.php [hunk 2]
```diff
@@ -2002,6 +2016,13 @@ function twitter_fetchparentposts(App $a, int $uid, $post, TwitterOAuth $connect
 	}
 }
 
+/**
+ * Fetches the posts received by the Twitter user
+ *
+ * @param App $a
+ * @param int $uid
```
Comment (Quix0r) [inline twitter/twitter.php:2024]: Same here, `@return void` won't hurt. ;-)

Comment (MrPetovan) [reply twitter/twitter.php:2024]: Done.

File: twitter/twitter.php (hunk 1 above)
Comment (MrPetovan) [reply twitter/twitter.php:1204]: Done.

**Pull request merged**